lblod/ember-rdfa-editor-lblod-plugins
# GN-4652: Start number

### Overview

* Allows to set a "start number" for a structure. 
* Introduce `numberDisplayStyle` and `startNumber` attributed for `_header` nodes.

##### connected issues and PRs:

https://binnenland.atlassian.net/browse/GN-4650


### Setup

1. Checkout
2. `npm run start`

### How to test/reproduce

Have a look at the video - https://share.cleanshot.com/d87pbryy

1. Try to reproduce what is happening in the video
2. Try checkout `master`, create a structure there, then checkout this branch and see if structure is still intact.

### Challenges/uncertainties

I have moved storing calculated `number` to the `content` attribute of "heading" element. So instead of storing "romanized" number, we now store actual integer (as string) and a `displayStyle` (on the root level). But, magically, this commit and current `master` and backwards compatible against each other, as we still store "converted" number (e.g. romanized) in the `textContent` of the "heading" element.  


### Checks PR readiness
